src/utilities/hopscotch_hash.h

```diff
@@ -47,13 +47,13 @@
 #include <string.h>
 #include <stdio.h>
 #include <limits.h>
-#include <math.h>
+//#include <math.h>
 
```

Comment:
I see you are commenting out various .h files. Are they no longer needed?

Reply:
Good question. I forgot the reason why I wanted to comment out math.h here. It should compile fine without, otherwise I will add it back.